inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -19,22 +21,103 @@
 SWASH_TEMPLATE_SUBDIR = "swash"
 SWASH_CONFIG_TEMPLATE_FILENAME = "input.sws.jinja"
 SWASH_CONFIG_FILENAME = "input.sws"
+SWASH_BATHYMETRY_FILENAME = "bathymetry.bot"
+
+
+class Bathymetry:
+    """Represents a bathymetric profile.
+    
+    A bathymetric profile defines the depth of the sea bottom as a function of
+    space, here described in Cartesian coordinates (x, y). Here, a bathymetry is
+    represented as a 2D array with the depths, in meters, at each point of a
+    regular grid. The grid is defined by the range of x and y values. Positive
+    depths are below the water level.
+    """
+
+    def __init__(
```

Comment:
maybe the constructor should be even simpler, and then you can either load or generate a random bathymetry.